src/client/_clientwindow.py

```diff
@@ -520,71 +502,62 @@ def resizeWidget(self, globalMousePos):
             self.setGeometry(newRect)
 
     def setup(self):
+        import news
         import chat
-        import tourneys
-        import stats
-        import vault
+        import coop
         import games
         import tutorials
-        import downloadManager
+        import stats
+        import tourneys
+        import vault
         import modvault
-        import coop
-        import news
+        import downloadManager
         from chat._avatarWidget import AvatarWidget
 
         # download manager
         self.downloader = downloadManager.downloadManager(self)
 
         self.loadSettings()
 
-        # Initialize chat
-        self.chat = chat.ChatWidget(self, self.players, self.me)
-
-        self.gameview_builder = GameViewBuilder(self.me,
```

Comment:
This stuff was done on purpose to conform to Python 80 character width limit. Autopep8 likes it, and it makes vertically split file editing so much easier :)

Reply:
if we really want to go PEP-8 .. way to go.  do we?

Reply:
I kinda like sticking to the 80 char limit, since vertical file split is a godsend. How about we leave existing code alone for now, and gradually move to 80 char limit as we refactor? It's the most minor of pep8 things anyway.

Reply:
ok, added 80 guideline to my Pycharm and reduced some.
Any idea why AppVeyor is failing?

Reply:
Travis says "QWidget: Must construct a QApplication before a QWidget", so I imagine reordering imports causes some module to create a widget before a QApplication is created. It's triggered somewhere in tests.

Reply:
How will I find that, Pycharm runs without a problem.

Reply:
Got the test runing.
I reversed the fix for: fa.instance.gameset = self.gameset  [_clientwindow, L155].
Seems the test didn't like the 'import client' in game_process.

Reply:
Did you verify that running games still works?

Reply:
yes